staging/src/k8s.io/apiserver/pkg/authentication/request/bearertoken/bearertoken.go

```diff
@@ -65,3 +53,21 @@ func (a *Authenticator) AuthenticateRequest(req *http.Request) (*authenticator.R
 
 	return resp, ok, err
 }
+
+// ExtractBearerToken gets the bearer token of a request, "ok" will be false if the token is missing
+func ExtractBearerToken(req *http.Request) (token string, ok bool) {
```

Comment:
Can you send this move as a separate PR? It's not good to mix refactoring and functionality change.